src/api/appHub.ts

```diff
@@ -0,0 +1,61 @@
+import { D2ApiGeneric } from "./d2Api";
+import { D2ApiResponse } from "./common";
+
+/* https://docs.dhis2.org/master/en/developer/html/webapi_app_store.html */
+
+export class AppHub {
+    constructor(public d2Api: D2ApiGeneric) {}
+
+    list(): D2ApiResponse<App[]> {
+        return this.d2Api.get<App[]>("/appHub");
+    }
+
+    install(version: string): D2ApiResponse<void> {
+        return this.d2Api.post(`/appHub/${version}`);
+    }
+}
+
+export interface App {
+    appType: string;
```

Comment:
[minor] Maybe we can narrow down the type: https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-api/src/main/java/org/hisp/dhis/appmanager/AppType.java